src/auro-datepicker.js

```diff
@@ -3,7 +3,7 @@
 
 // ---------------------------------------------------------------------
 
-/* eslint-disable max-lines, max-depth */
+/* eslint-disable max-lines, max-depth, no-magic-numbers, complexity, no-undef-init */
 
 // If using litElement base class
 import { LitElement, html } from "lit-element";
```

Comment:
This is importing the legacy Lit Element?

Reply:
Good catch, this is updated now.

Reply:
Good catch, this is updated now.